pandas/core/dtypes/generic.py

```diff
@@ -1,5 +1,12 @@
 """ define generic base classes for pandas objects """
 
+from typing import TYPE_CHECKING, Type, cast
+
+if TYPE_CHECKING:
+    from pandas.core.frame import DataFrame
```

Comment:
in TYPE_CHECKING block no harm in importing from top level

Reply:
Done.